docs/architecture/introduction.rst

```diff
@@ -265,8 +265,9 @@ Following the diagram, the execution would be:
    workloads (agents waiting to be scheduled).
 
 #. The master calculates the ideal size of the cluster and decides how many agents to launch and
-   which agents to terminate. The calculation is done based on the configured scaling behavior and
-   the specification of the resource pools.
+   which agents to terminate. The calculation is based on the configured scaling behavior (minimum
+   and maximum amount of instances the master can spawn) and the specification of the resource
+   pools.
```

Comment:
should we call out this calculation takes into account the total resource requests of all workloads? I am not sure it's implied or obvious.

>The calculation is based on the total resource requests of all jobs submitted to the cluster, configured scaling behavior (minimum and maximum amount of instances the master can spawn) and the specification of the resource pools.

Reply:
resolved